vecst.mod

```diff
@@ -342,12 +343,12 @@ static double findx (void* vv) {
   }
   nx=av[0]; // size of source vecs
   for (i=0;i<num;i++) { 
-    bv[i]=list_vector_px2(ob2, i, &bvo[i], &vv); // dest vectors 
-    if (vector_buffer_size(vv)<ni) { 
-      printf("findx ****ERRD**** arg#%d need:%d sz:%d\n",num+i+1,ni,vector_buffer_size(vv));
+    bv[i]=list_vector_px2(ob2, i, &bvo[i], &vv_tmp); // dest vectors 
```

Comment:
That's one for @iomaganaris -- personally I wasn't too worried about the `vector` methods in these ModelDB PRs, I haven't been trying to fix all the deprecation warnings (due to using `void*` prototypes instead of `IvocVect*` prototypes), just trying to fix the errors in C++.

Reply:
I think I got this change from some other model and I didn't pay too much attention to this since there was no warning or error. I fixed in though now so we don't have to declare a new variable